content/api/suppression-list.apib

```diff
@@ -4,48 +4,84 @@ description: Manage your suppression list - a list of recipient email addresses
 
-## Recipient Database Maintenance
+## Recipient Maintenance
 
-<banner status="danger">When initially configuring your SparkPost account, we <strong>strongly recommend</strong> you import any suppression list you have from any previous service to avoid incorrectly sending mail to unsubscribed/invalid recipients.</banner>
+It's good practice to maintain your recipient lists by removing recipients based on the bounce, unsubscribe and spam complaint events provided by SparkPost.
+These events are available from [webhooks](/api/webhooks), [message events](/api/message-events) and your suppression list.
+A 24-hour time-windowed suppression list search [as outlined below](#suppression-list-search-get) is useful here.
 
-<banner status="info">SparkPost supports bulk importing or manually adding up to 1,000,000 suppression list entries total.</banner>
+SparkPost supports bulk importing or manually adding up to 1,000,000 suppression list entries total.
 
-It is also good practice to maintain your own recipient database by unsubscribing or removing recipients based on the bounce, unsubscribe and spam complaint events provided by SparkPost. These events are available from [webhooks](webhooks), [message events](message-events) and your suppression list. A 24-hour time-windowed suppression list search [as outlined below](#suppression-list-search-get) is useful here.
+## Subaccount Suppressions
 
-## List Entry Attributes
-| Field         | Type     | Description                           | Required   | Notes   |
-|------------------------|:-:       |---------------------------------------|-------------|--------|
-| recipient | string | Email address to be suppressed | yes | |
-| type | string | Type of suppression record: `transactional` or `non_transactional` |  |
-|transactional | boolean | Whether the recipient requested to not receive any transactional messages | Not required if a valid `type` is passed | Available, but deprecated in favor of `type`|
-|non_transactional | boolean | Whether the recipient requested to not receive any non-transactional messages | Not required if a valid `type` is passed | Available, but deprecated in favor of `type` |
-|source | string | Source responsible for inserting the list entry. Valid values include: `Spam Complaint`, `List Unsubscribe`, `Bounce Rule`, `Unsubscribe Link`, `Manually Added`, `Compliance`| no - entries created by the user are marked as `Manually Added` | Field is read-only  |
-| description | string | Short explanation of the suppression | no | |
+Each subaccount has it's own independent suppression list.
+Set the `X-MSYS-SUBACOOUNT` header to a subaccount's ID to search and operate on that subuaccount's suppresion list.
+By default, the search endpoint works on all lists in the account (Master account and subaccount lists).
```

Comment:
Replace with something like "By default, the search endpoint will search across the master account's and all subaccounts'  lists." We should avoid parentheses - they are a crutch.